### PR TITLE
[YUNIKORN-992] website update for 1.0.0 release

### PR DESCRIPTION
**Pull request by wilfred-s**
Updated downloads page
Fixes in release procedure layout and roadmap update
New versioned doc for 1.0.0
New release-announce index page
Redirect for community page to get_involved

### CODE REVIEW

File: src/pages/community/download.md [hunk 1]
```diff
@@ -29,24 +29,25 @@ The downloads for the latest release are distributed via mirror sites.
 Older releases can be downloaded from the Apache archive site.
 All release artifacts should be checked for tampering using GPG or SHA-512.
 
-The latest release of Apache YuniKorn is v0.12.2.
+We publish pre-build docker images for everyone's convenience.
```
Comment (yuchaoran2011) [inline src/pages/community/download.md:32]: pre-built?

File: src/pages/release-announce/1.0.0.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+With the changes made to the helm charts running a helm upgrade from an older version to 1.0.0 might not work.
+The workaround is to first uninstall the old version and then perform a clean installation of the current release.
+
+### Configuration changes
+Size modifiers are now supported for the queue configuration. 
+This changes the way quota values are interpreted in the configuration.
+Before 1.0.0 all memory values were interpreted as _megabytes_ and vcore (cpu) values as _milli cores_.
+From 1.0.0 onwards memory values are specified in **bytes** and vcores are specified in **cores** [YUNIKORN-629](https://issues.apache.org/jira/browse/YUNIKORN-629).
+
+To maintain the same interpretation of values in the configuration make the following changes to the pre 1.0.0 configuration:
```
Comment (yuchaoran2011) [inline src/pages/release-announce/1.0.0.md:50]:
```suggestion
To maintain the same interpretation of values in the configuration make the following changes to the pre-1.0.0 configuration:
```

File: src/pages/release-announce/1.0.0.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+| memory | add M  | memory: 2048 | memory: 2048M |
+| vcore  | add m  | vcore: 250   | vcore: 250m   |
+| other  | none   | license: 1   | license: 1    | 
+
+### Resource Display and Communication
+The conversion of memory resources of pods and other objects by the k8shim into megabytes could cause a discrepancy between the size accounted for by the Kubernetes tools and YuniKorn.
+This could lead to issues around node fits and autoscaling of nodes.
+In 1.0.0 all memory resources are converted to **bytes** before being used in calculation and communications [YUNIKORN-1105](https://issues.apache.org/jira/browse/YUNIKORN-1105).
+
+Resources in all REST response objects are no longer a single string but are JSON structures using a key value pair for each resource type.
```
Comment (yuchaoran2011) [inline src/pages/release-announce/1.0.0.md:63]:
```suggestion
Resources in all REST response objects are no longer a single string but are JSON structures using a key-value pair for each resource type.
```

File: src/pages/release-announce/1.0.0.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+- `/ws/v1/queues` replaced with `/ws/v1/partition/{partitionName}/queues`
+- `/ws/v1/apps` replaced with `ws/v1/partition/{partitionName}/queues/{queueName}/application`
+- `/ws/v1/nodes` replaced with `/ws/v1/partition/{partitionName}/nodes`
+
+[YUNIKORN-954](https://issues.apache.org/jira/browse/YUNIKORN-954)
+
+## Highlights
+
+### Plugin mode technical Preview
+As part of this release a new deployment mode is added which is build on top of the Scheduler Plugin Framework.
```
Comment (yuchaoran2011) [inline src/pages/release-announce/1.0.0.md:93]:
```suggestion
As part of this release a new deployment mode is added which is built on top of the [Scheduling Framework](https://kubernetes.io/docs/concepts/scheduling-eviction/scheduling-framework/).
```

File: src/pages/release-announce/1.0.0.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+This is a major change in the way the scheduler k8shim works. As the implementation is not considered mature as yet the deployment mode is considered a tech preview.
```
Comment (yuchaoran2011) [inline src/pages/release-announce/1.0.0.md:94]:
```suggestion
This is a major change in the way the scheduler k8shim works. As the implementation is not considered mature yet the deployment mode is considered a tech preview.
```

**Review by yuchaoran2011 — APPROVED** [review]
LGTM except for some nits

File: src/pages/community/download.md (hunk 1 above)
Comment (wilfred-s) [reply src/pages/community/download.md:32]: prebuilt as one word according to Cambridge dictionary, will fix on commit.

**Pull request closed without merging**